README.md

```diff
@@ -1,13 +1,25 @@
 # Login Demo for iOS
 
-This is a demo application to demonstrate how to use Firebase's authentication in
-your iOS app.
+This is a demo of using Firebase's authentication features in
+your iOS app. It focuses on using OAuth with Google, Facebook, or Twitter. Note that
+Firebase also supports authentication with email & password and custom auth tokens.
+You can read the full [iOS authentication guide here](https://www.firebase.com/docs/ios/guide/user-auth.html).
 
-To download and setup all necessary SDKs, please run `pod install`. This requires that
-[Cocoapods](https://cocoapods.org/) is installed. 
+This demo requires that [Cocoapods](https://cocoapods.org/) is installed.
 
-You will need to create your own Firebase app and [enable the different
-authentication
-methods](https://www.firebase.com/docs/ios/guide/user-auth.html#section-providers)
-used in this app and update the values in `Login
-Demo/ViewController.m`.
+Running the Demo
+----------------
+
+To download and setup all necessary SDKs, run:
+
+    pod install
+
+Next, open `Login Demo.xcworkspace` in XCode (not `Login Demo.xcodeproj`,
+since you need to include the Cocoapod dependencies).
+
+You'll then need to edit the file `Login Demo/ViewController.m` and specify the
+Firebase app you're using as well as your Twitter / Google / Facebook OAuth API key.
```

Comment:
Facebook keys actually need to be put into `Supporting Files/Info.plist`. I would also call out needing to expand `URL types` since there's a field deep in there that also needs to get set for Facebook.